lib/rollbax.ex

```diff
@@ -84,9 +85,9 @@ defmodule Rollbax do
     end
 
     if config[:enable_crash_reports] do
-      # We do this because the handler will read `:reporters` out of the app's environment.
-      Application.put_env(:rollbax, :reporters, config[:reporters])
-      :error_logger.add_report_handler(Rollbax.Logger)
+      :logger.add_handler(:rollbax_handler, Rollbax.LoggerHandler, %{
+        reporters: config[:reporters]
+      })
```

Comment:
Adds our new custom handler to `:logger` using `add_handler/3`

[add_handler(HandlerId, Module, Config)](https://www.erlang.org/doc/apps/kernel/logger.html#add_handler/3)